OmniKit/Model/FaultEventCode.swift

```diff
@@ -117,197 +117,199 @@ public struct FaultEventCode: CustomStringConvertible, Equatable {
     }
     
     public var description: String {
+        let prefix = String(format: LocalizedString("Fault Event Code %1$02x:", comment:" Fault Event Code 0xXX"), rawValue)
+
         if let faultType = faultType {
             switch faultType {
             case .noFaults:
-                return LocalizedString("Fault Event Type 0x00: No faults, system running normally.", comment: "Fault Event Type 0x00: No faults, system running normally.")
+                return "\(prefix) No faults, system running normally"
```

Comment:
```
let faultDescription: String
if let faultType = faultType {
    faultDescription = { 
        switch faultType {
        case .noFaults:
            return "No faults, system running normally"
        ....
        }
    }()
} else {
    faultDescription = "Unknown Error Code"
}
return String(format: "Fault Event Code %1$02x: %@", rawValue, faultDescription)
```

Reply:
I was also looking for a way to return a combined string below, but I did not know where to add the return string variable. So right before the switch, it seems so logical now but I could not find any examples of this. Thanks!